### ISSUE TITLE
report_full_output

### ISSUE THREAD

**Issue by ghost** [walter-cd/walter#122]
Hello,
I noticed that in function ReportStageResult (in pipelines/pipeline.go) there is a self.Reporter.Post on the second line of the function. This means that all stages will output to the messenger no matter how you config the pipeline.yml file. For myself, I want to control what is being sent out to my Slack client, especially if the build file has many stages. My suggestion is to remove this line or add another configuration command to control this behavior (which seems like overkill).

Nice project!

**Comment by takahi-i** [comment]
Thank you very much for the comment @jim-edmiston! We will look into the function.

**Comment by ainoya** [comment]
>  I want to control what is being sent out to my Slack client, especially if the build file has many stages.

You're right, I think it's good to add some options to overkill or abbreviate or limit output of stage.

> Nice project!

:blush:

**Comment by takahi-i** [comment]
I will add a option *suppress* to suppress specified outputs.

```YAML
messenger:
  type: slack
  channel: $SLACK_CHANNEL
  username: $SLACK_USER_NAME
  url: $SLACK_URL
  suppress: result, stdout, stderr
```

Users add output types which they want suppress the results. The above setting suppress all the results from the stages (result, stdout, stderr).

**Comment by takahi-i** [comment]
Fixed with #125.

**Comment by takahi-i** [comment]
@ghost Please reopen this issue or create a new one if you are not satisfied with the feature.
